mkosi/__init__.py

```diff
@@ -3057,9 +3090,16 @@ def reuse_cache(context: Context) -> bool:
     return True
 
 
-def save_uki_components(
+def save_efi_components(
```

Comment:
The renaming seemed a bit weird to me upon first glance, but I understand why with a separate file you don't want to call it a UKI component. EFI component still sounds a bit weird to me, maybe ESP component?